webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-association.js

```diff
@@ -102,6 +104,17 @@ class SakaiRubricAssociation extends RubricsElement {
                   <input @change="${this.updateStateDetails}" name="rbcs-config-hideStudentPreview" type="checkbox" ?checked=${this.selectedConfigOptions["hideStudentPreview"]} value="1" ?disabled=${!this.isAssociated || this.readOnly}>${this.hideStudentPreview}
                 </label>
               </div>
+            ${this.showSelfReportCheck === 'true' ?
+              html`
+                <div class="checkbox">
+                   <label>
+                     <input @change="${this.updateStateDetails}" id="rbcs-config-" name="rbcs-config-studentSelfReport" type="checkbox" ?checked=${this.selectedConfigOptions["studentSelfReport"]} value="1" ?disabled=${!this.isAssociated || this.readOnly}>${this.studentSelfReport}
```

Comment:
You don't need the surrounding quotes on handler attachments.
@change=${this.updateStateDetails}

Reply:
Why does the id end with a hyphen and nothing else?

Reply:
Amended.